magical creatures/Hippogriff.java

```diff
@@ -0,0 +1,26 @@
+public class Hippogriff extends Creature{
+
+    public String species() {
+        return "Hippogriff"; 
+    }
+
+    public String description () {
+        return " A magical beast that has the front legs, wings, and head of an eagle and the body, hind legs, and tail of a horse. Hippogriffs are carnivores and are extremely dangerous to be tamed. \nStrengths: quick reactions, loyal, independent \nWeaknesses: a bit too independent and have a hard time trusting others";
+    }
+
+
+    public int health() {
+        return 220;
+    }
+
+    public int strength() {
+        return 20;
+    }
+
+
+    public String bestEnvir() {
```

Comment:
... then you use the enum here like this:
```suggestion
    public Environment.envirTypes bestEnvir() {
```